qiskit/circuit/controlflow/if_else.py

```diff
@@ -142,6 +143,16 @@ def c_if(self, classical, val):
             "Please nest it in an IfElseOp instead."
         )
 
+    def copy_no_body(self):
+        """Return a copy with body arguments replaced by empty circuits with
+        the intention that it will be populated by the caller."""
+        true_body = QuantumCircuit(self.params[0].qubits, self.params[0].clbits)
+        if self.params[1]:
+            false_body = QuantumCircuit(self.params[1].qubits, self.params[1].clbits)
+        else:
+            false_body = None
+        return IfElseOp(copy(self.condition), true_body, false_body=false_body, label=self.label)
```

Comment:
Similarly here, `copy.copy` is a shallow copy and `self.condition` is a tuple, so is immutable at that level.